Example/ViewController.swift

```diff
@@ -86,13 +88,18 @@ class ViewController: AsyncMessagesViewController, ASCollectionDelegate {
                 sender: sender)
             messages.append(message)
         }
-        dataSource.collectionView(collectionView: collectionView, insertMessages: messages, completion: nil)
+        dataSource.collectionNode(collectionNode: asyncCollectionNode, insertMessages: messages, completion: nil)
     }
     
-    func changeCurrentUser() {
+    @objc func changeCurrentUser() {
         let otherUsers = users.filter({$0.ID != self.dataSource.currentUserID()})
         let newUser = otherUsers[Int(arc4random_uniform(UInt32(otherUsers.count)))]
-        dataSource.collectionView(collectionView: collectionView, updateCurrentUserID: newUser.ID)
+        dataSource.collectionNode(collectionNode: asyncCollectionNode, updateCurrentUserID: newUser.ID)
+    }
+    
+    func collectionNode(_ collectionNode: ASCollectionNode, constrainedSizeForItemAt indexPath: IndexPath) -> ASSizeRange {
```

Comment:
I know that I suggested these lines, but after thinking again, I think we should provide a default delegate (similar to the existing default data source) that has this implementation by default. That way it'll be easier for new users to bootstrap the library (i.e don't require them to implement this method).

Reply:
Done